ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/ProjectAssignment/Index.cshtml

```diff
@@ -0,0 +1,63 @@
+﻿@page "/project-assignment/{id:int}"
+@using ApplyToBecomeInternal.Utils
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@model ApplyToBecomeInternal.Pages.ProjectAssignment.IndexModel
+@{
+    ViewData["Title"] = "Who will be on this project?";
+}
+
+@section BeforeMain
+    {
+    <a asp-page="@Links.TaskList.Index.Page" asp-route-id="@Model.Id" class="govuk-back-link">Back</a>
+}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            <span class="govuk-caption-l">@Model.SchoolName</span>
+            <h1 class="govuk-heading-l">Who will be on this project?</h1>
+
+            <div class="govuk-form-group">
+                <label class="govuk-label" for="delivery-officer">
+                    Delivery officer
+                </label>
+
+                <div id="delivery-officer-assigned-hint" class="govuk-hint">
+                    The delivery officer responsible for this project
+                </div>
+
+                <form method="post">
+                    <input type="hidden" asp-for="@Model.Id" />
+                    <select name="SelectedName" class="autocomplete__menu govuk-select" id="delivery-officer">
+                        <option value="@Model.SelectedDeliveryOfficer">@Model.SelectedDeliveryOfficer</option>
+                        @{
+                        foreach (var deliveryOfficer in Model.DeliveryOfficers)
+                        {
+                            <option value="@deliveryOfficer.FullName">@deliveryOfficer.FullName</option>
+                        }
+                    }
+                </select>
+
+                <div class="govuk-button-group govuk-!-margin-top-5">
+                    <button class="govuk-button" data-module="govuk-button">
+                        Continue
+                    </button>
+                    <input type="hidden" name="UnassignDeliveryOfficer" id="UnassignDeliveryOfficer" />
+                    <span class="govuk-body govuk-!-margin-right-2 govuk-!-margin-left-1">or </span><a href="#" id="unassign-link" class="govuk-link">Unassign the delivery officer</a>
```

Comment:
Will it having an empty href (href="#") ever be an issue?

Reply:
This was an awkward one - because the link is actually acting as a submit button.
I had to use js to make the button submit when clicked.
Its fairly standard to use # when no link is given - https://stackoverflow.com/questions/5637969/is-an-empty-href-valid

Reply:
Ok, cheers.